dev-docs/solr-source-code.adoc

```diff
@@ -0,0 +1,76 @@
+# Working With Solr Source Code
+
+## Building Solr from Source
+
+Download the Java 11 JDK (Java Development Kit) or later.  We recommend the OpenJDK
+distribution Eclipse Temurin available from https://adoptium.net/.
+You will need the JDK installed, and the $JAVA_HOME/bin (Windows: %JAVA_HOME%\bin)
+folder included on your command path. To test this, issue a "java -version" command
+from your shell (command prompt) and verify that the Java version is 11 or later. See https://github.com/apache/solr/blob/main/dev-docs/jvms.adoc[here] for more information on Gradle and JVMs.
+
+Download the Apache Solr distribution, from https://solr.apache.org/downloads.html.
+Unzip the distribution to a folder of your choice, e.g. C:\solr or ~/solr
+Alternately, you can obtain a copy of the latest Apache Solr source code
```

Comment:
Should not git be the first option we mention, and "Alternatively you can download the source..."?